spec/js/views/todo_view_spec.js

```diff
@@ -40,4 +39,33 @@ define([
       expect(instance.$el.find('.delete').length).toBe(expectedLength);
     });
   });
+
+  describe('Check item', function () {
+    beforeEach(function () {
+      subject = function () {
+        instance.render();
+        instance.toggleCompleteStatus();
+      };
+    });
+
+    it('executes call of function', function () {
+      var spy = spyOn(instance, 'toggleCompleteStatus').and.callThrough();
```

Comment:
Because you hold 'bussiness logic' in your model now you spy on it like you would spy on a service.